README.md

```diff
@@ -7,9 +7,9 @@ You can include bazelrio in your project like so:
 ```
 http_archive(
     name = "bazelrio",
-    url = "https://github.com/connorworley/bazelrio/archive/0c91c40c0aedf4bf9c5c78cefb92dc8301add16d.zip",
-    sha256 = "7e9b01e38097a1be1a43ae5d56e9215aa790fa13a81c92ef930ed67ceccdb5ab",
-    strip_prefix = "bazelrio-0c91c40c0aedf4bf9c5c78cefb92dc8301add16d/bazelrio",
+    url = "https://github.com/connorworley/bazelrio/archive/69366c248eb0d2592739e5d57a352860e56b7335.zip",
```

Comment:
Nit: Seems like this snuck in. She'll get updated whenever we cut a tag, so I don't care that much